dom.bs

```diff
@@ -8606,7 +8606,7 @@ associated <a>attribute</a>'s <a for=Attr>local name</a>.
 <var>token</var> are:
 
 <ol>
- <li><p>If the associated attribute's <a for=Attr>local name</a> does not define <a>supported
+ <li><p>If the associated <a>attribute</a>'s <a for=Attr>local name</a> does not define <a>supported
  tokens</a>, return true.
```

Comment:
Last nit, no breaking inside of a single element. (HTML does it, but try to avoid it as it makes search & replace harder than it needs to be.)

With that fixed, if you rebase/squash, I'll land this.